setup.py

```diff
@@ -34,19 +34,7 @@
     'maintainer_email': 'nathan@xanadu.ai',
     'url': 'https://github.com/XanaduAI/pennylane',
     'license': 'Apache License 2.0',
-    'packages': [
-                    'pennylane',
-                    'pennylane.beta',
-                    'pennylane.beta.plugins',
-                    'pennylane.circuit_drawer',
-                    'pennylane.interfaces',
-                    'pennylane.ops',
-                    'pennylane.optimize',
-                    'pennylane.plugins',
-                    'pennylane.qnodes',
-                    'pennylane.templates',
-                    'pennylane.vqe',
-                ],
+    'packages': find_packages(where="."),
     'entry_points': {
```

Comment:
Just curious, is this for ease of maintenance?

Reply:
Yep! A lot of bugs/build failures have been from us forgetting to include new subpackages here

Reply:
Now, new folders will be added automatically. The onus has now shifted the other way - if we ever create a folder that _shouldn't_ be a package, we will have to remember to exclude it here.